client/app/widgets/QuickCreate/__tests__/QuickCreate.spec.jsx

```diff
@@ -0,0 +1,179 @@
+import React from 'react';
+import axios from 'axios';
+import {
+  render,
+  screen,
+  getNodeText,
+  waitForElementToBeRemoved,
+} from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import {
+  QuickCreate,
+  sortAlpha,
+  labelExists,
+  addToCheckboxes,
+} from 'widgets/QuickCreate';
+
+const checkboxes = [
+  { id: 'last', value: 'a_value', label: 'zoo_label' },
+  { id: 'middle', value: 'middle_value', label: 'middle_label' },
+  { id: 'first', value: 'z_value', label: 'alpha_label' },
+  { id: 'dm', value: 'duplicate_middle_value', label: 'middle_label' },
+];
+
+// https://testing-library.com/docs/dom-testing-library/api-events#fireeventeventname
+// from rtl docs Most projects have a few use cases for fireEvent, but the majority of the time
+// you should probably use @testing-library/user-event.
+describe('QuickCreate', () => {
+  beforeEach(() => {
+    render(
+      <QuickCreate
+        name="name"
+        id="123"
+        label="label"
+        checkboxes={checkboxes}
+        formProps={{
+          inputs: [
+            {
+              id: 'category_name',
+              label: 'Name',
+              name: 'category[name]',
+              required: true,
+              type: 'text',
+              value: '',
+            },
+            {
+              id: 'submit',
+              name: 'submit',
+              type: 'submit',
+              value: 'Submit',
+            },
+          ],
+          action: 'https://if-me.org/quick-create',
+        }}
+      />,
+    );
+  });
+
+  afterEach(() => {
+    jest.restoreAllMocks();
+  });
```

Comment:
No problem. 

Maybe it would be worth exploring an eslint rule/plugin for a few reasons. One, new people like myself :) don't know what _looks good_ to a project. Two, I like having things prettier and eslint fix things for me. Three, having set rules yields less friction in a PR. 

Maybe something like this https://github.com/dangreenisrael/eslint-plugin-jest-formatting ? After this PR I kinda want to explore making the testing feedback loop a bit faster so maybe I could propose that along with anything I may find to speed up the tests.